src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -122,6 +122,21 @@ const FIELD_TYPE_TEXT_LINE = 'text_line';
 const FIELD_TYPE_TIME = 'time';
 const FIELD_TYPE_URL = 'url';
 
+const router = new Router(createHashHistory());
+
+router.addUpdateRouteHook((newRoute, newAttributes) => {
+    if (newAttributes.locale) {
+        router.setContext('locale', newAttributes.locale);
+        userStore.updateContentLocale(newAttributes.locale);
+    }
+
+    if (newAttributes.webspace) {
+        router.setContext('webspace', newAttributes.webspace);
```

Comment:
I don't think that this is necessary, because the webspace is already correctly remembered. Also, I would prefer not to couple the `AdminBundle` to the locale or webspace...

Reply:
sounds reasonable 👍